src/rt/sched/cown.h

```diff
@@ -967,16 +963,108 @@ namespace verona::rt
       return false;
     }
 
+    inline bool check_unmute_message(Alloc* alloc, MessageBody* msg)
+    {
+      if (msg->behaviour != nullptr)
```

Comment:
The Behaviour `nullptr` means `mute message`.

Reply:
yes